diff --git a/packages/geb-contract-api/src/generated/OracleRelayer.ts b/packages/geb-contract-api/src/generated/OracleRelayer.ts
index 95b551a..57f62bf 100644
--- a/packages/geb-contract-api/src/generated/OracleRelayer.ts
+++ b/packages/geb-contract-api/src/generated/OracleRelayer.ts
@@ -113,6 +113,14 @@ export class OracleRelayer extends BaseContractAPI {
         return this.ethSend(abi, [])
     }
 
+    redemptionPrice_readOnly(): Promise<BigNumber> {
+        // prettier-ignore
+        // @ts-ignore
+        const abi = {"inputs":[],"name":"redemptionPrice","outputs":[{"internalType":"uint256","name":"","type":"uint256"}],"stateMutability":"nonpayable","type":"function"}
+
+        return this.ethCall(abi, [])
+    }
+
     redemptionPriceUpdateTime(): Promise<BigNumber> {
         // prettier-ignore
         // @ts-ignore
